site/content/docs/runtimes/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: How runtimes work
+weight: 1
+menu:
+  docs:
+    parent: "Runtimes"
+---
+
+Checkly allows you to use JavaScript code in your [Browser checks](/docs/browser-checks) and in the optional 
+[setup & teardown scripts](/docs/api-checks/setup-teardown-scripts/) you can add to your API checks.
+
+This JavaScript code executes in a runtime environment that is managed by Checkly. This environment has access to specific 
+Node.js versions and NPM packages, like the "latest Playwright version on Node 14". We call these environments **runtimes**.
+
+## Using runtimes
+
+It's pretty easy. You don't have to do anything most of the time.
+
+1. View all the available runtimes on the [runtimes tab in the account section](https://app.checklyhq.com/account/runtimes). See the screenshot below.
+2. There is always one runtime marked as **active**. This is the runtime all checks use, unless you decide to override this at the check or group level.
+4. Runtimes have a simple `YYYY.MM` name, e.g. `2021.06` Newer dates hold newer dependencies.   
+3. Pick the runtime you want and click "Save settings". This runtime now applies to all of your checks and any new checks you create.
+
+![runtimes](/docs/images/monitoring/account_runtimes@2x.png)
+
+Note that one runtime version, e.g. `2021.6` holds all dependencies for all check types. There is no separate runtime for
+different types of checks.
+
+## Overriding runtimes for specific checks
+
+Before updating your Checkly account to a new runtime, you can enable a newer runtime just for one check or check group.
+This is a very powerful way to make check if your checks are compatible with a new runtime.
+
+You can make use of this by providing a default value for a specific variable at the global or group level, but allow that variable to
```

Comment:
This might be a case of The Friday, but I can't understand how this relates to this context. It might be clear for someone who already has an understanding of how runtimes work and are selected, but without that it wasn't for me. Can we elaborate on this, or give an example?

Reply:
lol, this was a copy and paste from some other section. Removed.